content/html/concepts/elements/terms/style/style.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: '<style>'
+Description: 'This element is used to apply a simple style sheet to an HTML document'
+Subjects:
+ - 'Web Development'
+ - 'Web Design'
+Tags: 
+ - 'Tags'
+ CatalogContent:
+ - 'learn-html'
+ - 'paths/front-end-engineer-career-path'
+---
+The **`<style>`** element is used to apply a styles to an HTML document
+
+## Syntax
+
+```html
+<!-- In-document style -->
+<style>
+  // CSS element goes here
+</style>
+
+<!-- Inline style -->
+<h1 style="color:yellow;text-align:center">This is a header</h1>
+<p style="color:red">This is a paragraph.</p>
+
+<!-- External style sheet -->
+<head>
+  <link rel="stylesheet" href="styles.css">
+</head>
```

Comment:
```suggestion
```

I would apply this information to a separate entry since this entry should focus solely on the `<style>` HTML element.